### PR TITLE
Reduce weight of heavy unit tests to evaluate K Drive

### PR DESCRIPTION
**Pull request by narasamdya**
[AB#1643778](https://dev.azure.com/mseng/708e929f-6bd5-415a-8daf-25b1dac08dd8/_workitems/edit/1643778)

### CODE REVIEW

File: Public/Src/Engine/UnitTests/FingerprintStore/Test.BuildXL.FingerprintStore.dsc [hunk 1]
```diff
@@ -36,6 +36,6 @@ namespace Test.BuildXL.FingerprintStore {
         runtimeContent: [
             importFrom("BuildXL.Utilities.UnitTests").TestProcess.deploymentDefinition
         ],
-        runTestArgs: { weight: 10 },
+        runTestArgs: { weight: 2 },
```
Comment (mpysson) [inline Public/Src/Engine/UnitTests/FingerprintStore/Test.BuildXL.FingerprintStore.dsc]: >  runTestArgs: { weight: 2 }, [](start = 6, length = 29)

Why not just remove them completely?

Comment (narasamdya) [reply Public/Src/Engine/UnitTests/FingerprintStore/Test.BuildXL.FingerprintStore.dsc]: Let's go with this weight first. If timeout doesn't happen, we can start reconsidering all weights that we set in our self-host.

**Pull request merged**